source/hpc.md

```diff
@@ -50,7 +50,7 @@ The scheduler doesn't need the whole node to itself so we can also start a worke
 
 ### Start Dask CUDA Workers
 
-Next start the other [dask-cuda workers](https://dask-cuda.readthedocs.io/). Dask-CUDA extends the traditional Dask `Worker` class with specific options and enhancements for GPU environments. Unlike the scheduler and client, the workers script should be _scalable_ and allow the users to tune how many workers are created.
+Next start the other [dask-cuda workers](https://docs.rapids.ai/api/dask-cuda/~~~rapids_api_docs_version~~~/). Dask-CUDA extends the traditional Dask `Worker` class with specific options and enhancements for GPU environments. Unlike the scheduler and client, the workers script should be \_scalable\* and allow the users to tune how many workers are created.
```

Comment:
Was the `\_scalable\*` intentional here? It [renders as `_scalable*` in the preview](https://rapids-deployment--403.org.readthedocs.build/en/403/hpc/#start-dask-cuda-workers).

Reply:
This was definitely **not** intentional, thanks for catching it.

When I was experimenting with other templating approaches and tested something like this:

```text
[dask-cuda workers](https://docs.rapids.ai/api/dask-cuda/{{ rapids_api_docs_version }}/
```

That space character and the use of `{{` together led one of the precommit hooks (I think `markdownlint`? I don't remember) to make some weird decisions about where that URL ended. And it added this weird escaping as a result.

Now that we're using template characters that are valid for URLs, and without spaces, that's not a problem 😁